ethereum/contracts/BasicInboundChannel.sol

```diff
@@ -80,11 +62,8 @@ contract BasicInboundChannel is InboundChannel {
             nonce = nonce + 1;
 
             // Deliver the message to the target
-            // Delivery will have fixed maximum gas allowed for the target app
             (bool success, ) =
-                _messages[i].target.call{value: 0, gas: MAX_GAS_PER_MESSAGE}(
-                    _messages[i].payload
-                );
```

Comment:
Maybe I'm missing something, but don't we still need this gas check? I think only the ones above with the `require` statements were unnecessary, right?

Reply:
Nevermind, has been added here again: https://github.com/Snowfork/polkadot-ethereum/pull/279